src/inspector/inspector.coffee

```diff
@@ -0,0 +1,13 @@
+import "inspector/polyfills/vendor/details-element-polyfill"
+import "inspector/element"
+import "inspector/global"
+import "inspector/control_element"
+#= require_tree ./templates
```

Comment:
What's this one left here for?

Reply:
This is the bit I still need to finish, the inspector templates don't work yet.